examples/presigned-getobject-request-date.js

```diff
@@ -27,8 +27,14 @@ var s3Client = new Minio.Client({
 })
 
 // Presigned get object URL for my-objectname at my-bucketname, it expires in 7 days by default.
-var requestDate = new Date('2017-01-01');
-var presignedUrl = s3Client.presignedGetObject('my-bucketname', 'my-objectname', 1000, {}, requestDate, function(e, presignedUrl) {
+var requestDate = new Date('2017-01-01')
```

Comment:
We don't need this old date. We can set `requestDate` to current date
```suggestion
var requestDate = new Date()
```